`kill -HUP` is valid in POSIX

we could extend the check to look for valid signal names, but that seems like
it's not worth the effort (when the # of signals is >30).  if the code uses a
wrong signal name, then it'll hit a runtime error regardless of shell.

https://sourceforge.net/p/checkbaskisms/checkbashisms/5/
http://pubs.opengroup.org/onlinepubs/9699919799/utilities/kill.html
https://bugs.gentoo.org/458392

--- a/checkbashisms
+++ b/checkbashisms
@@ -496,7 +496,6 @@
 	qr'\&>' =>	               q<should be \>word 2\>&1>,
 	qr'(<\&|>\&)\s*((-|\d+)[^\s;|)}`&\\\\]|[^-\d\s]+(?<!\$)(?!\d))' =>	
 				       q<should be \>word 2\>&1>,
-	$LEADIN . qr'kill\s+-[^sl]\w*' => q<kill -[0-9] or -[A-Z]>,
 	$LEADIN . qr'trap\s+["\']?.*["\']?\s+.*[1-9]' => q<trap with signal numbers>,
 	$LEADIN . qr'trap\s+["\']?.*["\']?\s+.*ERR' => q<trap ERR>,
 	qr'\[\[(?!:)' => q<alternative test command ([[ foo ]] should be [ foo ])>,
